source/platforms/bitbucket_server/BitBucketServerAPI.ts

```diff
@@ -213,7 +213,7 @@ export class BitBucketServerAPI implements BitBucketServerAPIDSL {
 
     return comments
       .filter(comment => comment!.text.includes(dangerIDMessage))
-      .filter(comment => username || comment!.author.name === username)
```

Comment:
I'm not sure the history behind the `username || ` part of this code but it does not seem like the proper logic? Happy to revert this if I need to.

Reply:
Yeah, I can't figure it out either, so lets take it and see